tests/new_unit_tests/Commands/Site/ImportCommandTest.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests\Commands\Connection;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+use Pantheon\Terminus\Commands\Connection\InfoCommand;
+use Pantheon\Terminus\Config;
+
+use Prophecy\Prophet;
+use Terminus\Models\Environment;
+use Terminus\Models\Site;
+use VCR\VCR;
+
+/**
+ * Test suite for class for Pantheon\Terminus\Commands\Connection\InfoCommand
+ */
+class InfoCommandTest extends ConnectionCommandTest
```

Comment:
As the author of `InfoCommandTest` I am flattered with the attribution but I think you can safely resume this work under a new title, say, `ImportCommandTest`. Namespace and friends too.